src/main/java/uk/gov/hmcts/reform/profilesync/service/impl/ProfileSyncServiceImpl.java

```diff
@@ -98,6 +97,11 @@ public String getS2sToken() {
         do {
             formParams.put("page", String.valueOf(counter));
             Response response = idamClient.getUserFeed(bearerToken, formParams);
+            log.info("Response code from idamClient.getUserFeed {}", response.status());
+            if (response.status() != 200 && response.body() != null) {
+                log.info("Response body from idamClient.getUserFeed {}", response.body().toString());
```

Comment:
log only if response is not 200. It should not print any PII data as the call has already failed.

Reply:
you can check response itself is not null otherwise could be npe at response.status()? 
put || in below condition , we want to logs if status not 200 and and even if body is null
if (response.status() != 200 && response.body() != null)

Reply:
can u also log the status code?

Reply:
Hi Shashank, the status code is being logged in previous line.